locales/zh-CN.json

```
@@ -435,8 +463,10 @@
     "edited": "在 {0} 编辑了"
   },
   "tab": {
+    "accounts": "帐户",
```

十分抱歉，我去考证了一下相关资料，发现先前的建议是错误的用词，还是保留原先的翻译吧

```suggestion
    "accounts": "账户",
```

---

我刚才也查了一下，貌似很多地方（如知乎）用的也是 `帐`

---

不如直接`用户` 好了 🤣 你觉得如何

---

我看了一下使用到这个文案的界面，我认为改成 `用户` 没问题。你也可以看看：https://deploy-preview-1571--elk-zone.netlify.app/universeodon.com/list/2768/accounts

---

我也觉得可以